index.html

```diff
@@ -0,0 +1,55 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="styles.css" />
+    <title>Document</title>
+  </head>
+
+  <body>
+    <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
+
+    <nav id="navbar">
+      <ul>
+        <li><a href="#welcome-section"> Welcome </a></li>
+        <li><a href="#projects">Work</a></li>
+        <li><a href="#contact">Contact</a></li>
+      </ul>
+    </nav>
+
+    <section id="welcome-section">
+      <h1>Welcome to my portfolio site</h1>
+    </section>
+
+    <section id="projects">
+      <h2>First project</h2>
+      <div id="project">
+        <div class="project-tile">First Project</div>
+        <a href="https://www.google.com">Link to my project</a>
+      </div>
+      <div id="project2">
+        <div class="project-tile">second Project</div>
+        <a href="https://www.google.com">Link to my project</a>
```

Comment:
It would be better to use less section tags.Section tags should only be used to distinguish different part of your lay out.